lib/ssl/src/tls_record.erl

```diff
@@ -568,7 +568,6 @@ validate_tls_record_length(Versions, {_,Size0,_} = Q0, MaxFragLen,
                     %% Complete record
                     {Fragment, Q} = binary_from_front(Length, Q0),
                     Record = #ssl_tls{type = Type, version = Version, fragment = Fragment},
-                    ssl_logger:debug(get(log_level), inbound, 'record', Record),
```

Comment:
We might want to do this during handshaking, or we will loose OpenSSL debug logging compatibility?!

Reply:
Keep this but Comment out the code and comment why it is outcommented.